src/settings.js

```diff
@@ -1771,7 +1771,7 @@ var MIN_IE_VERSION = 0x7FFFFFFF;
 // Edge 44.17763 was released on November 13, 2018
 // MAX_INT (0x7FFFFFFF, or -1) specifies that target is not supported.
 // [link]
-var MIN_EDGE_VERSION = 44;
+var MIN_EDGE_VERSION = 0x7FFFFFFF;
```

Comment:
I think this is still wrong, Edge definitely should be supported by default.

Reply:
Do you understand this this setting does not refer to the current Edge browser (which is based on chromium) but the many years old EdgeHTML engine which does exist anymore?

Reply:
No I don't. When I wrote this code, I fully intended it to refer to Edge as a product, there is no reason to limit its meaning to old EdgeHTML. The version numbering they devised for old vs new Edge conveniently follows a scheme that can be utilized here. There is no reason not to.

Reply:
If we want to continue to refer to the modern Edge browser then I suppose we would want to add an Edge column to the  https://webassembly.org/roadmap/ page?

Reply:
I guess I can use this map to go from edge version to chrome version: https://en.wikipedia.org/wiki/Microsoft_Edge#New_Edge_release_history

Reply:
Do the first chromium-based Edge release was based on chrome 79 it seems: https://en.wikipedia.org/wiki/Microsoft_Edge#New_Edge_release_history.  And from then on the edge version numbers pretty much match up with chromium version numbers.   Given that, does it still make sense to have two different version numbers?

Reply:
Yes, certainly, no trouble dropping support for old EdgeHTML itself.

There is a valid question though how 1:1 the new Edge features are with Chromium features. I don't actually know.

We could argue for the case that if new Edge is 1:1 with their version numbers and features with Chromium version numbers and features, it would not make sense to duplicate these version fields over to two options, and, say,  `MIN_CHROME_VERSION=100` and `MIN_EDGE_VERSION=100`.

One approach to move forward might be to add a comment along the lines of 
```
// For now, this field refers to supporting old EdgeHTML browser, but if discrepancies between Chromium-based Edge and Chrome surface, then this field can be updated to refer to new Edge options as well.
```
so we'd keep this field frozen and reinstate when/if it becomes useful.

Reply:
Sounds like a plan

Reply:
OK, I added to comment to make it clear the this currently refers to EdgeHTML but this might change if modern-Edge and Chromium start to diverge.